bundle/manifests/nodetopology.openshift.io_numaresourcesoperators.yaml

```diff
@@ -179,6 +179,55 @@ spec:
                       type: string
                   type: object
                 type: array
+              machineconfigpools:
```

Comment:
hm looks like rebasing issue

Reply:
probably this should be handled by a separate PR, on it

Reply:
https://github.com/openshift-kni/numaresources-operator/pull/41